cl/search/signals.py

```diff
@@ -0,0 +1,81 @@
+from cl.lib.custom_es_signal_processor import ESSignalProcessor
+from cl.search.documents import ParentheticalGroupDocument
+from cl.search.models import (
+    Citation,
+    Docket,
+    Opinion,
+    OpinionCluster,
+    OpinionsCited,
+    Parenthetical,
+    ParentheticalGroup,
+)
+
+pa_fields_mapping = {
```

Comment:
Some documentation of this dict would be really helpful. Maybe a comment first, explaining what the keys/values are, and some comments within it, explaining anything else that's interesting?

Reply:
Sure, I've added the documentation and some comments within the dict.